pandas/core/internals/concat.py

```diff
@@ -25,6 +25,7 @@
 
 import pandas.core.algorithms as algos
 from pandas.core.arrays import DatetimeArray, ExtensionArray
+from pandas.core.arrays.sparse.dtype import SparseDtype
```

Comment:
i'm not sure if we have a policy, but in general if the type is imported just for type checking, we normally put in an `if TYPE_CHECKING:` block. If this is not creating import errors then maybe OK. see what others think.

Reply:
@simonjayhawkins, I updated imports as you suggested, so that SparseDtype is used only for type checking.